### DISCUSSION TITLE
Black images are back

### DISCUSSION THREAD

**Discussion by Tillerz** [vladmandic/sdnext#1050] [Q&A]
Did something change with rendering since yesterday? I suddenly get black images sometimes again, where I had none for over a week. And it surely isn't a memory problem. `Time taken: 18.93s | GPU active 5688 MB reserved 13914 MB | System peak 15598 MB total 24564 MB`

It's not happening very often, though.

**Comment by vladmandic** [comment]
no changes that should have material impact on this.

**Comment by TheOnlyHolyMoly** [comment]
@Tillerz: I have had the same issue with some models and got it solved with adjusting model precsion in settings. for some reason some models only work when I change model precision into the one direction and others only when changing into the other direction. Have not investigated further since this allowed me getting it fixed through change of settings.  Maybe this will work for you too.

**Reply by Tillerz** [reply]
Ok, will fiddle around. It didn't happen for a long time, then like 3 times within 5 minutes, and now it's gone again. I can live with it if it happens that rarely. :)
